recipes/image-meta-tag/bld.bat

```diff
@@ -0,0 +1 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Please remove the `bld.bat` and `build.sh` in favor of the `script` tag in the `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26